README.md

```diff
@@ -1,11 +1,82 @@
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
+
 # Aragon JS SDK
 
-This folder contains all the JS packages available on NPM:
+Welcome to the Aragon JS SDK! This SDK allows you to interact with Aragon OSx
+using JavaScript. Whether you are a developer wanting to build on top of Aragon,
+or looking to contribute to the SDK itself, you are in the right place! This
+folder contains all the JS packages available on NPM:
 
 - [Client](./modules/client)
 - [IPFS](./modules/ipfs)
 - [Common](./modules/common)
 
+## Developer Quick Start
+
+The Aragon OSx SDK is Javascript agnostic, which means you can use it with any
+Javascript framework, including popular ones like React, Vite, or Vue. The
+examples in this readme are written using NodeJS
```

Comment:
Let's use "README" instead of "readme" for consistency.

Also, let's add a `.` at the end of the sentence.